ixmp/backend/base.py

```diff
@@ -902,8 +1020,8 @@ def get_meta(self, s: Scenario):
         """
 
     @abstractmethod
-    def set_meta(self, s: Scenario, name_or_dict, value=None):
-        """Set single or multiple meta entries.
+    def set_scenario_meta(self, s: Scenario, name_or_dict, value=None):
```

Comment:
o be duplicated after renaming

Reply:
with that one https://github.com/iiasa/ixmp/pull/353/files#diff-313b0713bfcf3262902e8b94639dd49eR986